services/server/config/config.go

```diff
@@ -203,6 +213,24 @@ func (bc *BoltConfig) Validate() error {
 	}
 }
 
+const (
+	// ExporterTypeOTLP represents the open telemetry exporter OTLP
+	ExporterTypeOTLP = "otlp"
+)
+
+// Validate openTelemetry config
```

Comment:
openTelemetry -> OpenTelemetry. Please double-check your change. The way you spell OpenTelemetry is still inconsistent.

Reply:
Changed the names to OpenTelemetry in all occurrences.